assets/public/js/main.js

```diff
@@ -6,3 +8,93 @@ function toggleCartDropdown() {
 }
 
 cartBtn.onclick = toggleCartDropdown;
+
+/* Navigation menus */
+
+const mainNavButtonId = "navBtn";
+const featuresSubNavButtonId = "featuresBtn";
+const backButtonId = "backBtn";
```

Comment:
If you use these consts only once in your code you might as well use the strings directly without creating consts

Reply:
Fixed.